.github/workflows/e2e-nightly-37x.yml

```diff
@@ -0,0 +1,104 @@
+# Runs randomly generated E2E testnets nightly on the v0.37.x branch.
+
+# !! This file should be kept in sync with the e2e-nightly-main.yml file,
+# modulo changes to the version labels.
+
+name: e2e-nightly-37x
+on:
+  schedule:
+    - cron: '0 2 * * *'
+
+jobs:
+  e2e-nightly-test:
+    # Run parallel jobs for the listed testnet groups (must match the
+    # ./build/generator -g flag)
+    strategy:
+      fail-fast: false
+      matrix:
+        group: ['00', '01', '02', '03', "04"]
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.18'
+
+      - uses: actions/checkout@v3
+        with:
+          ref: 'v0.37.x'
+
+      - name: Capture git repo info
+        id: git-info
+        run: |
+          echo "::set-output name=branch::`git branch --show-current`"
+          echo "::set-output name=commit::`git rev-parse HEAD`"
```

Comment:
I can't find this part in yml for `main`, intended?

Reply:
Yes. This is one of the parts that need to be taken from the `34x` file to facilitate our Slack failure notifications.

With `main` it's easy because the latest commit hash and branch details are provided by GitHub through environment variables when the nightly E2E tests run.

But because we manually check out the `v0.34.x` / `v0.37.x` branches, we need to store this information in output variables during the test execution job so we can use them later during the notification jobs. If we don't do this, and we rely on those environment variables, we just get `main` and the latest commit hash on `main`, because that's where the nightly run was triggered.